docker/docker-push.sh

```
@@ -2,14 +1,36 @@
+if [ -z "$1" ]; then
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)